authors/apps/articles/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from .utils import generate_slug, unique_code_generator
+from django.db.models.signals import pre_save
+from authors.apps.profiles.models import Profile
+from django.contrib.postgres.fields import ArrayField
+
+
+class Tag(models.Model):
+    tag_name = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.tag_name
+
+
+class Article(models.Model):
+    title = models.CharField(max_length=100)
+    slug = models.SlugField(unique=True, blank=True)
+    description = models.CharField(max_length=100)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True, null=True)
+    favorited = models.BooleanField(default=False)
+    favoritesCount = models.IntegerField(default=0)
+    body = models.TextField()
+    post_images = ArrayField(models.URLField(max_length=500), default=list)
```

Comment:
@mariamiah  multiple image support nice

Reply:
<img width="553" alt="screen shot 2019-01-21 at 12 15 40 pm" src="https://user-images.githubusercontent.com/39744587/51464342-62a77980-1d76-11e9-94ba-281834c5ee16.png">

changed it to that